db/ecmcDS.substitutions

```diff
@@ -1,4 +1,4 @@
-file ecmcDS.template
+file ecmcDS-idx.template
```

Comment:
the 'x' should be 'X' in an effort to make it similar to the chX we have for the DAQ?